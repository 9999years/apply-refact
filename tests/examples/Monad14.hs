yes = do forM files $ \x -> return (); return ()