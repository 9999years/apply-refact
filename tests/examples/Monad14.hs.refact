[("tests/examples/Monad14.hs:1:10-37: Warning: Use forM_\nFound:\n  forM files $ \\ x -> return ()\nPerhaps:\n  forM_ files $ \\ x -> return ()\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 10, endLine = 1, endCol = 14}, subts = [], orig = "forM_"}])]
