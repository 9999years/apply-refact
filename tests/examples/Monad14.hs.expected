yes = do forM_ files $ \x -> return (); return ()
